input/tar2/goconvey.goconvey

```diff
@@ -0,0 +1,2 @@
+#-timeout=30s
+-short
```

Comment:
Are you sure you want this file included? I didn't see tests with a short marker.